src/index.js

```diff
@@ -21,27 +23,42 @@ const tasksList = [
 ];
 
 const createTask = (task) => {
+  const divContainer = document.createElement('div');
   const li = document.createElement('li');
+  const checkValue = (task.completed === true) ? 'checked' : '';
+  const checkClass = (task.completed === true) ? 'marked' : '';
+  divContainer.classList.add('div-container');
+  li.classList.add('task-item');
 
   li.innerHTML = `
-    <li class="task-item">
-      <label class="task-label">
-        <input type="checkbox" value="${task.completed}">
-        <p class="task-description">${task.description}</p>
-      </label>
-    <i class="fas fa-ellipsis-v"></i>
-    </li>`;
```

Comment:
- [ ] Kindly fix the add functionality for your app, when I enter a task and press the enter key, it doesn't add it. Please look into this and fix it

Reply:
>     * [ ]  Kindly fix the add functionality for your app, when I enter a task and press the enter key, it doesn't add it. Please look into this and fix it

Thanks for your review. However, I think that's part of the next exercise requirements. The requirements for this task are available here https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md. Please let me know if it's neccesary.